### PR TITLE
Debug and Stabilize Logging and Loki

### PR DESCRIPTION
**Pull request by Just-en-Time**
Updates to increase default resource allocations for loki and logging agents, increase buffer and batch for logging. Should review before implementing in general prod, tested in smaller but high traffic prod environments

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,7 +24,8 @@ else
 endif
 
 WITHOUT_JOBS := $(shell echo $(ALL_MODULES) | tr ' ' '\n' | grep -v -e "nomad" | awk '{print "-target=module." $$0 ""}' | xargs)
-ALL_MODULES_ARGS := $(shell echo $(ALL_MODULES) | tr ' ' '\n' | awk '{print "-target=module." $$0 ""}' | xargs)
+# TEMPORARILY DISABLED FOR DEBUG ALL_MODULES_ARGS := $(shell echo $(ALL_MODULES) | tr ' ' '\n' | awk '{print "-target=module." $$0 ""}' | xargs)
+ALL_MODULES_ARGS := -target=module.nomad.nomad_job.loki -target=module.nomad.nomad_job.logs-collector
```
Comment (ValentaTomas) [inline Makefile]: Let's not forget to later remove this debugging edit done for the deploy convenience.

File: packages/nomad/logs-collector.hcl [hunk 2]
```diff
@@ -136,21 +136,22 @@ teamID = "{{ teamID }}"
...
-auth.user = "${var.grafana_logs_username}"
-auth.password = "${var.grafana_api_key}"
-
-[sinks.grafana.labels]
-source = "logs-collector"
-service = "{{ service }}"
-teamID = "{{ teamID }}"
-envID = "{{ envID }}"
-sandboxID = "{{ sandboxID }}"
+# [sinks.grafana]
```
Comment (ValentaTomas) [inline packages/nomad/logs-collector.hcl]: We might probably also cleanup the vars that were passed here from TF that might no longer be used before this goes out of draft.

File: Makefile (hunk 1 above)
Comment (justine-time) [reply Makefile]: I just added a `make debug-apply` make target to keep both and restore `make apply` to its original purpose

File: packages/nomad/logs-collector.hcl (hunk 2 above)
Comment (justine-time) [reply packages/nomad/logs-collector.hcl]: Good call, cleaned up

**Comment by jakubno** [comment]
Important parts merged to `dev`

**Pull request closed without merging**